exercises/phone-number/phone_number_test.exs

```diff
@@ -38,6 +38,16 @@ defmodule PhoneTest do
   end
 
   @tag :pending
+  test "invalid when proper number of digits but letters mixed in and length is 10" do
+    assert Phone.number("1a2a3a4a5a") == "0000000000"
+  end
+
+  @tag :pending
+  test "invalid when proper number of digits but letters mixed in and length is 11" do
```

Comment:
Sorry to be so questioning about this, but do you think it's possible that something could pass the test above but fail this test? We've been doing a lot of cleanup of the test suites recently, removing redundant tests, and I worry this might be one of them.